plenum/server/view_change/view_changer.py

```diff
@@ -644,17 +643,17 @@ def _start_selection(self):
 
         self.provider.select_primaries()
 
```

Comment:
I believe we need to call `node.on_view_change_complete` (at least do some of the actions there).